ActionSheet.android.js

```diff
@@ -87,19 +96,24 @@ class ActionGroup extends React.Component {
     );
 
     for (let i = startIndex; i < startIndex + length; i++) {
-      let color = '#212121';
-      if (i === destructiveButtonIndex) {
-        color = '#d32f2f';
-      }
-
+      const defaultColor = tintColor
+        ? tintColor
+        : (textStyle || {}).color || BLACK_87PC_TRANSPARENT;
+      const color = i === destructiveButtonIndex ? '#d32f2f' : defaultColor;
+      const iconSource = icons != null && icons[i];
       let iconElement = undefined;
 
-      if (icons && icons[i]) {
-        const iconStyle = [styles.icon]
-        if (textStyle.color !== undefined && textStyle.color !== null) {
-          iconStyle.push({ tintColor: textStyle.color })
+      if (iconSource) {
+        if (typeof iconSource === 'number') {
+          const iconStyle = [styles.icon, { tintColor: tintIcons ? color : null }];
```

Comment:
what does tintIcons have to do with setting the color here

This is how it reads:
If `tintIcons` exist, set it to color. Otherwise set it to null?

Reply:
Well `tintIcons` is a Boolean prop you can set if you want to apply whatever colour the text will be to the icons as well. If someone doesn’t want to change the tint (ie they’ve exported the icons at a specific colour they want already, they can set this to false.

Maybe a different name for this prop instead?

Reply:
need to know if the name needs to be better, or is fine as-is. I did add all of the new ones to the documentation.

Reply:
Seems clear to me with the added documentation.